SQL_assignment/sql_assignment1

```diff
@@ -0,0 +1,82 @@
+CREATE TABLE: Creates an Employees table with columns for EmployeeID, FirstName, LastName, Department, and Salary.
+
+    Create table emp(EmpoyeeID int primary key,firstname varchar(20),lastname varchar(20),department varchar(20),salery int)
+
+
+
+INSERT: Adds three employee records to the table.
+
+    insert into emp values(1,'akash','j','IT',50000)
+    insert into emp values(,2'arun','p','NON-IT',32000)
+    insert into emp values(31,'Abhishek','s','IT'38000)
+
+
+SELECT: Shows different ways to query data:
+
+    select* from emp
+    select Department,FirstName from emp where EmployeeID=1
+    select Firstname ,salary from emp
+
+
+Select all columns and rows
+  
+    select* from emp
+
+
+Select specific columns
+
+    select Firstname ,salary from emp
+
+
+Select with a WHERE clause to filter results
+
+    select Department,FirstName from emp where EmployeeID=1
+
+
+What is the purpose of the IDENTITY keyword in the CREATE TABLE statement?
+
+    IDENTITY keyword is used to auto increment columns
+    IDENTITY(1,1) means it will start with 1 and when each data is enterd its value will be increased by 1
```

Comment:
What is meant by IDENITITY(5,2) ?

Reply:
the first value will be 5 and when the next row is added the value will be autoincremented to 7.
that is it starts with 5 and 2 is incremented each step